pylops/jaxoperator.py

```diff
@@ -0,0 +1,19 @@
+__all__ = [
+    "JaxOperator",
+]
+
+
+import jax
+
+from pylops import LinearOperator
+
+
+class JaxOperator(LinearOperator):
+    def __init__(self, Op):
+        super().__init__(dtype=Op.dtype, dims=Op.dims, dimsd=Op.dimsd, name=Op.name)
```

Comment:
if you put a comma at the end it will be more readable

Reply:
not sure what you mean...